src/appfl/agent/server.py

```diff
@@ -128,6 +128,7 @@ def set_sample_size(
             sample_size: int,
             sync: bool = False,
             blocking: bool = False,
+            **kwargs
```

Comment:
What do we do with this?

Reply:
In some place, it is called via `ret_val = self.server_agent.set_sample_size(client_id, **meta_data)`, so just in case `meta_data` contains some additional parameters